master/internal/trial.go

```diff
@@ -739,9 +728,6 @@ func (t *trial) sendNextWorkload(ctx *actor.Context) error {
 		t.TerminationSent = true
 		actors.NotifyAfter(ctx, terminateTimeoutPeriod, terminateTimeout{runID: t.RunID})
 	} else {
-		if err := saveWorkload(t.db, w); err != nil {
```

Comment:
seems like a kinda weird in-between state to be in where workloads are getting pushed to the rest api to be persisted into the steps table but also pushed back through the websocket for the workload sequencer to coordinate next steps with the trial. do you think we need a feature branch for the second half of this (moving the workload sequencer) and this to be lumped together into a single change? or maybe wait until you know you can hit them both in a single release?

Reply:
Maybe we can talk about this offline.

Reply:
>do you think we need a feature branch for the second half of this (moving the workload sequencer) and this to be lumped together into a single change? or maybe wait until you know you can hit them both in a single release?

We don't need a feature branch since it won't break anything. It won't be weird after removing the trial workload sequencer entirely from the master. I don't want to do all the changes at once.

Reply:
I understand not doing them all at once. I think the approach for that is good. Just want to make sure no in between parts fall through the cracks.